pkg/cli/debug_recover_loss_of_quorum_test.go

```diff
@@ -408,14 +408,16 @@ func TestUpdatePlanVsClusterDiff(t *testing.T) {
 	for _, d := range []struct {
 		name    string
 		nodes   []int
+		restart []int
```

Comment:
nit: consider `staleLease` or some such, because we'll implicitly also be restarting other nodes.